src/components/app/routes/queries/subsidies/subscriptions.js

```diff
@@ -12,29 +12,44 @@ async function fetchSubscriptionLicensesForUser(enterpriseUUID) {
   return camelCaseObject(response.data);
 }
 
-async function fetchCustomerAgreementData(enterpriseUUID) {
-  const queryParams = new URLSearchParams({
-    enterprise_customer_uuid: enterpriseUUID,
+const transformSubscriptionLicenseResponse = (response) => {
+  // Deep copies the response from learner-licenses
+  const subscriptionLicenses = structuredClone(response);
+  // Create and assign the customer agreement object
+  const customerAgreement = {
+    results: [response.results[0].customerAgreement],
```

Comment:
nit: maybe check for non-empty results here?

Reply:
The newly updated paginator now returns a null value for an empty customer agreement, so the API is handling the case where an enterprise does not have a customer agreement.